source/faq.txt

```diff
@@ -64,63 +64,6 @@ Why am I seeing a warning about a non-genuine MongoDB server?
 
 .. include:: /includes/fact-non-genuine-warning.rst
 
-.. _compass-faq-sampling:
-
-What is sampling and why is it used?
-------------------------------------
-
-Sampling in |compass| is the selection a subset of data
-from a particular collection and analyzing the documents within the
-sample set.
-
-Sampling is a common technique in statistical analysis because analyzing
-a subset of the data gives similar results to analyzing all of it. In
-addition, sampling allows results to be generated quickly rather than
-performing a computationally-expensive collection scan.
-
-How does sampling work?
------------------------
-
-|compass| employs two distinct sampling mechanisms.
-
-In MongoDB 3.2, collections are sampled with the
-:manual:`$sample </reference/operator/aggregation/sample/>` operator via
-the :manual:`aggregation pipeline </core/aggregation-pipeline>`. This
-provides efficient random sampling without replacement over the entire
-collection, or over the subset of documents specified by a query.
-
-.. _compass_fallback_sampling:
-
-In MongoDB 3.0, collections are sampled via a
-backwards-compatible algorithm executed entirely within |compass|. It
-takes place in three stages:
-
-1. |compass| opens a :term:`cursor` on the desired collection, limited
-   to at most 10,000 documents sorted in descending order of the ``_id``
-   field.
-2. ``sampleSize`` documents are randomly selected from the stream. To
-   do this efficiently, |compass| employs `reservoir sampling
-   <http://en.wikipedia.org/wiki/Reservoir_sampling>`_.
-3. |compass| performs a query to select the chosen documents directly
-   via ``_id``.
-
-``sampleSize`` is set to 1000 documents.
-
-.. note::
-  The choice of sampling method is done transparently in the
-  background, with no changes required by the user.
-
-Won't sampling miss documents?
```

Comment:
looks like this section didn't make it to the new page

Reply:
It did initially, but after rewriting I felt like the key info was included in the other paragraphs. Sampling is explained and justified, and this section just goes on to say the MongoDB team tries to find a balance which didn't seem necessary. I'm open to adding something like this section back in.

Reply:
sounds good to me